R/tm_g_heat_bygrade.R

```diff
@@ -7,27 +7,27 @@
 #'
 #' @inheritParams teal.widgets::standard_layout
 #' @inheritParams argument_convention
-#' @param sl_dataname (\code{character}) subject level dataset name,
-#' needs to be available in the list passed to the \code{data}
-#' argument of \code{\link[teal]{init}}
-#' @param ex_dataname (\code{character}) exposures dataset name,
-#' needs to be available in the list passed to the \code{data}
-#' argument of \code{\link[teal]{init}} \cr
-#' @param ae_dataname (\code{character}) adverse events dataset name,
-#' needs to be available in the list passed to the \code{data}
-#' argument of \code{\link[teal]{init}} \cr
-#' @param cm_dataname (\code{character}) concomitant medications dataset name,
-#' needs to be available in the list passed to the \code{data}
-#' argument of \code{\link[teal]{init}} \cr
-#' specify to \code{NA} if no concomitant medications data is available
-#' @param id_var (\code{choices_seleced}) unique subject ID variable
-#' @param visit_var (\code{choices_seleced}) analysis visit variable
-#' @param ongo_var (\code{choices_seleced}) study ongoing status variable,
-#' This variable is a derived logical variable. Usually it can be derived from \code{EOSSTT}.
-#' @param anno_var (\code{choices_seleced}) annotation variable
-#' @param heat_var (\code{choices_seleced}) heatmap variable
-#' @param conmed_var (\code{choices_seleced}) concomitant medications variable,
-#' specify to \code{NA} if no concomitant medications data is available
+#' @param sl_dataname (`character`) subject level dataset name,
+#' needs to be available in the list passed to the `data`
+#' argument of [teal::init()]
+#' @param ex_dataname (`character`) exposures dataset name,
+#' needs to be available in the list passed to the `data`
+#' argument of [teal::init()] \cr
+#' @param ae_dataname (`character`) adverse events dataset name,
+#' needs to be available in the list passed to the `data`
+#' argument of [teal::init()] \cr
+#' @param cm_dataname (`character`) concomitant medications dataset name,
+#' needs to be available in the list passed to the `data`
+#' argument of [teal::init()] \cr
+#' specify to `NA` if no concomitant medications data is available
+#' @param id_var (`choices_seleced`) unique subject ID variable
+#' @param visit_var (`choices_seleced`) analysis visit variable
+#' @param ongo_var (`choices_seleced`) study ongoing status variable,
```

Comment:
```suggestion
#' @param ongo_var (`choices_seleced`) study ongoing status variable.
```

Reply:
@kartikeyakirar no need to create a separate commit to approve this https://github.com/insightsengineering/teal.osprey/pull/260/commits/589902cbb6dd248dbbf63883587f0b7a0c8ac727

You can always `Sign off and commit suggestion` to accept this suggestion directly from GitHub.

<img width="727" alt="image" src="https://github.com/insightsengineering/teal.osprey/assets/133694481/413cc9eb-8a19-4324-923a-daa4c2c9eb78">


The only motivation that I see to create a commit instead of accepting a suggestion is to have a documentation be rendered. But you did not render the documentation in your commit.

Reply:
> You can always Sign off and commit suggestion to accept this suggestion directly from GitHub.

I did same. thats why on this commit, you were shown as a co-author . on separate commit Github doesnt' show.
![image](https://github.com/insightsengineering/teal.osprey/assets/6700955/709322e5-0221-458d-b05c-d1017bb37c77)

Reply:
coolio. my bad